### DISCUSSION TITLE
More personalization with custom theming

### DISCUSSION THREAD

**Discussion by AmadeusGraves** [Kovah/LinkAce#524] [Ideas & Feature Requests]
The application could have the possibility to change to more colors than the 3 default themes, it would be more attractive because you could create more eye-catching and pleasing to the eye themes.

**Comment by Kovah** [comment]
Thanks for your idea!
This is probably not a thing that will be included in version 2 which I am currently working on.
However, it is possible to add your own CSS to style the pages. Not a real solution, but a workaround.

For reference:
- #437
